stable/database/templates/statefulset.yaml

```diff
@@ -495,8 +526,39 @@ spec:
           # the SM becomes unready if it does not start within 15 minutes = 30s + 15s*58
           successThreshold: 2
           timeoutSeconds: {{ default 5 .Values.database.sm.readinessTimeoutSeconds }}
+{{- if and .Values.database.insights .Values.insights.enabled }}
```

Comment:
It looks like this section is repeated 4 times in the database chart. Can we make a template helper from that?

Reply:
If you are refering to the if template helper does not add a lot of value.  If you are referring to the whole inclusion,  probably.  I expect it is the same fpr all.

Reply:
I have a build that puts the whole section into a template.   One of the issues is that it depends upon log-volume which is not named the same between SM and TE.  should change the volume name in the te also to be log-volume instead of log-te-volume.  Then it's good.  I made changes by adding a _sidecar.tpl that has 2 templates.   One for additional volumes and one for additional sidecar containers.   That way there is one place if we add more sidecars.   But I don't know how to push these changes to this PR. Or if we want the changes, or if we want a different PR.

Reply:
You can just push the changes to the same branch `dbutson/sidecar` and they will appear in the PR.